File: 1823-find-the-winner-of-the-circular-game/1823-find-the-winner-of-the-circular-game.cs
public class Solution {
    public int FindTheWinner(int n, int k) {
        List<int> friends = new List<int>();
        
        for (int i = 1; i <= n; i++)
        {
            friends.Add(i);
        }
        
        int currentPosition = 0;
        
        while (friends.Count > 1)
        {
            currentPosition = (currentPosition + k - 1) % friends.Count;
            friends.RemoveAt(currentPosition);
        }
        
        return friends[0];
    }
}

File: 1823-find-the-winner-of-the-circular-game/1823-find-the-winner-of-the-circular-game.cpp
class Solution {
public:
    int findTheWinner(int n, int k) {
        vector<int>v;
        for(int i=1; i<=n; i++)
        {
            v.push_back(i);
        }
        
        int currentPosition = 0;
        
        while(v.size() > 1)
        {
            currentPosition = (currentPosition + k - 1) % v.size();
            v.erase(v.begin() + currentPosition);
        }
        
        return v[0];
    }
};